docs/reference/search-application/apis/delete-search-application.asciidoc

```diff
@@ -0,0 +1,47 @@
+[role="xpack"]
+[[delete-search-application]]
+= Delete Search Application API
+[subs="attributes"]
+++++
+
+<titleabbrev>Delete Search Application</titleabbrev>
+
+Removes a Search Application and its associated alias.
+Indices attached to the Search Application are not removed.
+
+[[delete-search-application-request]]
+== {api-request-title}
+
+`DELETE _application/search_application/<name>`
+
+[[delete-search-application-prereq]]
+== {api-prereq-title}
+
+Requires the `manage_search_application` cluster privilege.
+Also requires manage `manage_index` privileges on all indices that are included in the Search Application.
```

Comment:
the index privilege is just called `manage`, not `manage_index` - there's another reference to `manage_index` in the PUT API docs

Reply:
Fixed, thank you!